### PR TITLE
Homepage partners section #3

### LINKED ISSUE #3: 🎨 HomePage Partners Section
#  🎨 Create the partner's section on the HomePage
---

Quick links: [Summary](#summary) | [Description](#description) | [Technical-Analysis](#Technical-Analysis) | [Acceptance-Criteria](#Acceptance-Criteria)

---

### Summary

As a user, I want a section that shows/lists all our core business partners. Check the full design below for section placement. Use the assets provided below.

partners-section

Please take note of all the design features

#### Link to full design:
```
https://xd.adobe.com/view/b98caf27-6549-49e5-8ed8-c5234788df61-ecdf/screen/cc35e19a-24f2-42ce-8803-ee2942839528/
```

#### Company Logos
VVOB:
```
https://res.cloudinary.com/dyl3rncv3/image/upload/v1681380406/elewa-group-website/client-logos/vvob-logo_qyvjbp.jpg
```

KSA:
```
https://res.cloudinary.com/dyl3rncv3/image/upload/v1681380406/elewa-group-website/client-logos/Kenya_Scouts_Association_kcoka5.svg
```

farmbetter:
```
https://res.cloudinary.com/dyl3rncv3/image/upload/v1681380406/elewa-group-website/client-logos/farmbetter-logo_x2acda.png
```
WWF:
```
https://res.cloudinary.com/dyl3rncv3/image/upload/v1681380406/elewa-group-website/client-logos/wwf-4-logo-png-transparent_tbkfcq.png
```

Enabel:
```
https://res.cloudinary.com/dyl3rncv3/image/upload/v1681380405/elewa-group-website/client-logos/Enabel-logo_afiel7.png
```

Spotlights:
```
https://res.cloudinary.com/dyl3rncv3/image/upload/v1681380412/elewa-group-website/client-logos/Spotlights_7_pplnvm.jpg
```

### Technical Analysis
This section should be present in the `index.html` file

### Acceptance Criteria
1. When rendered the component is pixel-perfect compared to the design above.
2. The component is clean and does not have more than 50 lines
3. The design is responsive for screen, tablet and phone

### PR DESCRIPTION
**Pull request by Michal379**
# Description

I added the partners images on the partners section and fixed their styling.

Fixes #3 

## Added images
##Added styling to the images

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

- Firmware version:
- Hardware:
- Toolchain:
- SDK:

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,19 +1,27 @@
...
...Hx8fHx8fHx8fHx8fHx8fHx8fHx8fHx8fHx8fHz/wAARCAA8AIIDASIAAhEBAxEB/8QAGgABAAMBAQEAAAAAAAAAAAAAAAMEBQIGAf/EADEQAAEEAgECBAQDCQAAAAAAAAEAAgMEBRESITETMkFhBlFx0RQVNCNCUmJyc7HB4f/EABUBAQEAAAAAAAAAAAAAAAAAAAAB/8QAFBEBAAAAAAAAAAAAAAAAAAAAAP/aAAwDAQACEQMRAD8A9giIoCIiAiIgIiICIiAiIginsRVmc5pGsb7lV48tSldxbYaD79FaliZMwslYHtPcEbXm8vhvwzTPW2Yv3m/w/wDEHpx1GwoxPEXcRIzl21yG1gslkq4uGDxHF8o5nZ8rfQBdYqqLE5c8bYzqR8z6IPQIiICIsn4hnlgrRGGR0ZL9EtOvRBrIsiHG3P2bzkZSOhI2VpuniZK2N0jWvcNhpPUhBIigiu1ppDHFPG949AV9ntQVtePKyPfYE9SgmRRwWIbDeUMjZB/Keyyadnw8teM83GNvbk7oOqDaRRQWYbLeUErZAO/E9kntQVgDPK1m+2z3QSouIpo52c4nte35tO1l4eR772QD3ucGyEAE9upQa6+EAggjYPcKnlbT61bcW+bjoHW9KvjrN2y8eJoRt7u4637IKGRINx7W9Gs00D5ALYxdcwVByGnPPIrivjGsmM0x5vJ3r0CvoCIiAsX4n/Sw/wBf+ltKhlaDshCxjZAwtdvZG0EdDHSwSRzOuSSNA8hHTt9VVy0TZ83Tifvi5ujo9xsqZuOyTdD8x6D04qzZoPmyde0HtDYhotPc9/ugzsnViqZKg+uwR836Ib0HQj7q5fsQm2yuKos2NbAOgAPqpMhQfbsVZWva0Qu5EH16j7Li7jpZLjbdScRTAaPIbBQUscHRZ6VhhbByj6sadj0SpWisZ254zA8NJIB7K5Sx1mO+63anY95bx01qjkxlxl2axUssZ4p6hzeyCvAxtX4kdFA3jG5vVo7DptRQSGfK25ZKrrJaeIA1po37/RamPxn4WV880pmsP7u1rSjsYyYW3WqM4hkf5muGwUEWNhniycrxWfBXkb5SRoFML+vyP90/5KvUorcZebc7Zd60Gt1pVm4yeC++xWsBrJXcnscPfaDUREQEREBERAREQEREBERAREQEREBERAREQEREBERB/9k=" alt="">
```
Comment (JAPHETHNYARANGA) [inline index.html:19]: these image links are not feasable to be used

**Review by JAPHETHNYARANGA — COMMENTED** [review]
is the section responsive?, i cannot see media queries

File: index.html (hunk 1 above)
Comment (Michal379) [reply index.html:19]: Yes the section is responsive on all screen sizes

Comment (JAPHETHNYARANGA) [reply index.html:19]: use the image links provided , if not there let me know and share screen shots, mobile and web. Thank you.

Comment (Michal379) [reply index.html:19]: Screenshot from 2023-10-04 15-33-47

**Comment by Michal379** [comment]
Screenshot from 2023-10-04 15-43-49

**Comment by JAPHETHNYARANGA** [comment]
very good work, but please add emoji to the pr name and check on the conflicts. Closing this pr since the issue is done

**Pull request closed without merging**